get-started/welcome.qmd

```diff
@@ -0,0 +1,108 @@
+---
+title: "Welcome to Shiny"
+editor:
+  markdown:
+    wrap: sentence
+---
+
+Shiny for Python is a powerful and beginner-friendly framework for building interactive web applications and dashboards, all in pure Python.
+Whether you’re a data scientist, analyst, or developer, Shiny makes it easy to create rich, interactive experiences without needing to learn JavaScript or front-end frameworks.
+
+## Batteries included: everything you need to build a dashboard {#batteries-included}
+
+Shiny for Python comes fully equipped with everything you need to build a dashboard right out of the box.
+It includes a rich set of [input and output components](/components) so you can quickly build interactive applications without worrying about external dependencies.
+Layout options let you organize your UI efficiently, while built-in theming (including dark mode) ensures your app looks great with minimal effort.
+You can even use a [brand.yml](https://posit-dev.github.io/brand-yml/) file to apply consistent branding, colors, and logos across your application.
+
+## Reactivity: the secret to seamless interactivity {#reactivity}
+
+At the heart of Shiny is [reactivity](../docs/reactive-foundations.qmd), a system that automatically updates outputs when inputs change.
+Unlike traditional web apps, where you need to manually handle state and data updates with callbacks, Shiny’s reactive engine keeps everything in sync effortlessly.
+For example, when a user interacts with a slider or selects a filter, reactivity figures out the **minimum** amount of calculations to update the outputs without requiring you to write complex event-handling code.
```

Comment:
It'd be nice if the example backed up this phrase a little better. Right now the example uses the value from the slider everywhere, but it could have

* two sliders
* one output that updates whenever either slider changes
* one output that updates only when _one_ of the sliders change

Also small nit, but instead of claiming that reactivity figures out the minimum amount of calculations (which makes it sounds like Shiny runs some kind of optimization engine), it's probably better to phrase it like "Shiny's reactive engine avoids uneccessary computations by only re-calculating the outputs whose inputs have changed".